src/cli/commands/build/build.ts

```diff
@@ -104,6 +107,12 @@ export async function build(options: SWACLIConfig) {
     logger.log(`Building api with ${chalk.green(apiBuildCommand)} in ${chalk.dim(apiLocation)} ...`);
     runCommand(apiBuildCommand, apiLocation!);
   }
+  const end = new Date().getTime();
+
+  collectTelemetryEvent(TELEMETRY_EVENTS.Build, {
+    appFramework: projectConfig?.name?.split(", with")[0]!,
```

Comment:
can we ensure that this code doesn't throw null reference

Reply:
will address in next iteration